dts/bindings/gpio/arduino-nano-header-r3.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022 Joylab AG
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    GPIO pins exposed on Arduino Nano (R3) headers.
+
+    The Arduino Nano layout provides two headers on opposite edges of the board.
+
+    * A 15-pin header with mostly digital signals. The additional NRST (pin3)
```

Comment:
In case you need to push one more time, please fix `The additional NRST (pin 3)`